src/generateControllers.js

```diff
@@ -11,11 +11,14 @@ function generateControllers({ herbarium, controller = defaultController }) {
         let controllerInfos = {
             entity: entity,
             name: item.group,
-            getAll: usecases.getAll ? { usecase: usecases.getAll, controller } : undefined,
-            getById: usecases.getById ? { usecase: usecases.getById, controller, id: (resourceId?.name || 'id') } : undefined,
-            post: usecases.post ? { usecase: usecases.post, controller } : undefined,
-            put: usecases.put ? { usecase: usecases.put, controller } : undefined,
-            delete: usecases.del ? { usecase: usecases.del, controller } : undefined
+            getAll: usecases.getAll ? { usecase: usecases.getAll.usecase, controller, REST: usecases.getAll.REST } : undefined,
+            getById: usecases.getById ? { usecase: usecases.getById.usecase, controller, REST: usecases.getById.REST, id: (resourceId?.name || 'id') } : undefined,
+            post: usecases.post ? { usecase: usecases.post.usecase, controller, REST: usecases.post.REST } : undefined,
+            put: usecases.put ? { usecase: usecases.put.usecase, controller, REST: usecases.put.REST } : undefined,
+            delete: usecases.del ? { usecase: usecases.del.usecase, controller, REST: usecases.del.REST } : undefined,
+            other: usecases.other.length ? usecases.other.map((usecase) => {
```

Comment:
It seems `controllerInfos` needs a `group` in order to work. What is a use case has no `group`? Ex: `GET /search - Search all entities in your domain`

Reply:
A usecase without a group will not have an endpoint, in this case.

Reply:
We need to think in a structure that deals with:
- [x] CRUD use cases - Clear action and clear resource.
- [ ] Extra use cases for a resource - need extra metadata (Ex: `herbarium.usecases.get('UpdateUserPermission').metadata({ REST: { path: '/user/:id/permission' } })`)
- [ ] Use cases not linked to a resource (ex: `/search`)

Here the resource = group, since the current convention uses the group as the `resourceName`.

Reply:
I'll take a look in this cases.

Reply:
Ok, now the code works if an usecase is created without linked to a resource (group and resourceName), but REST.path should be passed.